src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCFIntegrationTest.java

```diff
@@ -701,4 +699,19 @@ public void testSpanDelRegression() throws IOException {
         //no more star alleles because deletions are all gone
         Assert.assertFalse(variants.stream().anyMatch(v -> v.getAlternateAlleles().contains(Allele.SPAN_DEL)));
     }
+
+    @Test
+    public void warnWhenRemovingAnnotations() {
```

Comment:
I would change this name since this doesn't actually warn.  Maybe "testRemoveAbsentAnnotation"?